### PR TITLE
LPS-128765 Make sure that all fragmentEntryLinks referenced in the layout structure are included. If any is not found, just add a dummy for it to avoid frontend errors

### PR DESCRIPTION
**Pull request by victorg1991**
I have tried several thing in the frontend, but they required to modified several files and also having to take into account this case in future developments. Instead of that I've moved the code to the backend to ensure that we always have a fragmentEntryLink, if that doesn't exist in the database, we'll show an error element that the user can remove

### CODE REVIEW

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by victorg1991** [comment]
ci:test:relevant

File: modules/apps/layout/layout-content-page-editor-web/src/main/java/com/liferay/layout/content/page/editor/web/internal/display/context/ContentPageEditorDisplayContext.java [hunk 1]
```diff
@@ -1573,6 +1573,35 @@ private JSONArray _getFragmentEntryLinkCommentsJSONArray(
 			themeDisplay.setIsolated(isolated);
 		}
 
+		LayoutStructure layoutStructure = _getLayoutStructure();
+
+		Map<Long, LayoutStructureItem> fragmentLayoutStructureItems =
+			layoutStructure.getFragmentLayoutStructureItems();
+
+		for (long fragmentEntryLinkId : fragmentLayoutStructureItems.keySet()) {
+			if (!fragmentEntryLinksMap.containsKey(
```
Comment (ealonso) [inline modules/apps/layout/layout-content-page-editor-web/src/main/java/com/liferay/layout/content/page/editor/web/internal/display/context/ContentPageEditorDisplayContext.java:1582]: If () 
 continue;

Is better in this case

Comment (ealonso) [reply modules/apps/layout/layout-content-page-editor-web/src/main/java/com/liferay/layout/content/page/editor/web/internal/display/context/ContentPageEditorDisplayContext.java:1582]: cc: @victorg1991

**Comment by ruben-pulido** [comment]
Just started reviewing :)

**Comment by ruben-pulido** [comment]
https://github.com/liferay-echo/liferay-portal/pull/4001

**Pull request closed without merging**